kart/docs.py

```diff
@@ -0,0 +1,183 @@
+"""
+This module provides functionality to
+write a man page from some given information
+about a CLI application and is inspired from
+click-contrib/click-man.
+
+:copyright: (c) 2016 by Timo Furrer.
```

Comment:
Maybe add your name to the copyright? I don't actually know what to do here
@rcoup probably knows about licenses

Reply:
if this file is copied from an online source, include the URL in the docstring here (ideally a github blob url with an OID in it so it's clear what exact version it was copied from). No need to adjust the copyright unless you've modified it substantially, in which case perhaps you need to say so here.

Either way the license for this file remains MIT not GPL. The MIT license actually requires the license text be included in distributed works, so at minimum it should be copied into this docstring I think.

Reply:
@rcoup can you double-check this? (license text has now been added to the docstring)